extensions/wikia/WallNotifications/tests/WallNotificationEntityTest.php

```diff
@@ -157,7 +185,7 @@ private function getWallOwnerMock( array $params ) {
 		return $wallOwner;
 	}
 
-	private function getArticleTitleMock( array $params ) {
+	private function getArticleTitleMock( array $params ): PHPUnit_Framework_MockObject_MockObject {
 		$articleTitle = $this->getMockBuilder( 'Title' )
 			->setMethods( [
 				'exists',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "exists" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)